sensors/accelerometer.h

```diff
@@ -0,0 +1,12 @@
+#ifndef ACCELEROMETER_H
+#define ACCELEROMETER_H
```

Comment:
This is more of a _by the way_ than a request for change. Instead of defining include guards like this, you can also simply use the snappier one liner `#pragma once`. Replaces 3 lines with 1!